docs/predict-api/v2/required_api.md

```diff
@@ -0,0 +1,829 @@
+# Predict Protocol - Version 2
+
+This document proposes an predict/inference API independent of any
+specific ML/DL framework and capable of supporting both easy-to-use
+and high-performance use cases. By implementing this protocol both
+inference clients and servers will increase their utility and
+portability by being able to operate seamlessly on platforms that have
+standardized around this API. This protocol is endorsed by NVIDIA
+TensorRT Inference Server, TensorFlow Serving, and ONNX Runtime
```

Comment:
@rakelkar can we get someone to approve ONNX compliance?

Reply:
Nathan Yan added "ONNX" endorsement to proposal doc, which I took to mean "ONNX Runtime"